README.md

```diff
@@ -201,184 +108,53 @@ You'll need to have a Spark release on your system and the `$SPARK_HOME` environ
+- [Cannoli](https://github.com/bigdatagenomics/cannoli) uses ADAM's [pipe](#pipes)
+  API to parallelize common single-node genomics tools (e.g.,
+  [BWA](https://github.com/lh3/bwa),
+  [bowtie2](http://bowtie-bio.sourceforge.net/bowtie2/index.shtml),
+  [FreeBayes](https://github.com/ekg/freebayes))
+- [DECA](https://github.com/bigdatagenomics/deca) is a reimplementation of the
+  XHMM copy number variant caller on top of ADAM/Apache Spark
+- [Gnocchi](https://github.com/bigdatagenomics/gnocchi) provides primitives for
+  running GWAS/eQTL tests on large genotype/phenotype datasets using ADAM
+- [Lime](https://github.com/bigdatagenomics/lime) provides a parallel
+  implementation of genomic set theoretic primitives using the [region join
+  API](#join)
+- [Mango](https://github.com/bigdatagenomics/mango) is a library for visualizing
+  large scale genomics data with interactive latencies and serving data using the
+  [GA4GH schemas](https://github.com/ga4gh/schemas)
 
-Our Avro schemas are directly converted into source code using Avro tools. Avro supports a number of computer languages. ADAM uses Java; you could 
-just as easily use this Avro IDL description as the basis for a Python project. Avro currently supports c, c++, csharp, java, javascript, php, python and ruby. 
 
-# Downstream Applications
+# Connecting with the ADAM team
 
-There are a number of projects built on ADAM, e.g.
+The best way to reach the ADAM team is to post in our [Gitter
+channel](https://gitter.im/bigdatagenomics/adam) or to open an issue on our
+[Github repository](https://github.com/bigdatagenomics/adam/issues). For more
+contact methods, please see [our support page](https://github.com/bigdatagenomics/adam/blob/master/SUPPORT.md).
 
- - [avocado](https://github.com/bigdatagenomics/avocado)  Variant caller
- - [cannoli](https://github.com/heuermh/cannoli)  Pipe API wrappers for bioinformatics tools
- - [cs-bwamem](https://github.com/ytchen0323/cloud-scale-bwamem)  Scalable alignment using bwamem
- - [eggo](https://github.com/bigdatagenomics/eggo)	 Parquet-formatted public 'omics datasets
- - [gnocchi](https://github.com/bigdatagenomics/gnocchi)	Genotype-phenotype analysis
- - [guacamole](https://github.com/hammerlab/guacamole)  Variant caller from Hammer Lab at Mt. Sinai
- - [mango](https://github.com/bigdatagenomics/mango)  Scalable genome visualization
- - [quinine](https://github.com/bigdatagenomics/quinine)  Read and variant quality control metrics
- - [rice](https://github.com/bigdatagenomics/rice)	 RNA quantification pipeline
- - [snappea](https://github.com/bigdatagenomics/snappea)	Parallel alignment using SNAP
 
 # License
 
-ADAM is released under an [Apache 2.0 license](LICENSE.txt).
+ADAM is released under the [Apache License, Version 2.0](LICENSE.txt).
 
-[Avro]: http://avro.apache.org
-[Spark]: https://spark.apache.org/
-[Parquet]: https://parquet.apache.org/
-[releases]: https://github.com/bigdatagenomics/adam/releases
 
 # Citing ADAM
 
```

Comment:
The homebrew section in README.md (which doesn't show up here in the diff) should probably be removed, I'll replace it later with new homebrew and conda sections.  Homebrew might take some work going forward, in that they go to JDK 9 by default which will break `apache-spark` and other upstream deps.

Reply:
JDK9. Nice. Aggressive.